subprojects/docs/src/snippets/dependencyManagement/dependencyLocking-lockingAllConfigurations/kotlin/build.gradle.kts

```
@@ -12,7 +12,7 @@ dependencyLocking {
 tasks.register("resolveAndLockAll") {
     notCompatibleWithConfigurationCache("Filters configurations at execution time")
     doFirst {
-        require(gradle.startParameter.isWriteDependencyLocks)
+        require(gradle.startParameter.isWriteDependencyLocks, "$path must be run from the command line with the `--write-locks` flag")
```

[`require()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/require.html) from the Kotlin stdlib uses a lazy message

```suggestion
        require(gradle.startParameter.isWriteDependencyLocks) { "$path must be run from the command line with the `--write-locks` flag") }
```

---

🤦 yes of course